public/javascripts/registration-validation.js

```diff
@@ -0,0 +1,61 @@
+var errorNames = [];
```

Comment:
@meedheym @ElliDy 
Do not define vars in global scope. 
If you don't want to use any build system, at least use wrapper or module pattern: 

```
(function(YourLib){

})(YourLib);
```